crates/lib/src/bootc_composefs/repo.rs

```diff
@@ -54,6 +54,25 @@ pub(crate) async fn initialize_composefs_repository(
     .await
 }
 
+/// skopeo (in composefs-rs) doesn't understand "registry:"
```

Comment:
Yeah...that was something I invented in ostree-ext and probably should have argued to add to the containers-transports so that podman/skopeo/etc understand.